src/main/webapp/WEB-INF/views/cars/add.jsp

```diff
@@ -8,6 +8,7 @@
     <title>Add car</title>
 </head>
 <body>
+<jsp:include page="/WEB-INF/views/header/header.jsp"/>
```

Comment:
Why do we need register or login when we have already logined?

Reply:
for the sake of logout